### PR TITLE
Git self sign

### PR DESCRIPTION
**Pull request by vparfonov**

### CODE REVIEW

**Comment by rkratky** [comment]
@vparfonov, I suggested some fixes in https://github.com/eclipse/che-docs/pull/939

File: src/main/pages/che-7/installation-guide/proc_deploying-che-with-support-for-git-repositories-with-self-signed-certificates.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
++
+In the command, substitute `_<ca.crt>_` for the self-signed certificate and _`<host:port>_` for the host and port of the HTTPS connection on the Git server (optional).
++
+NOTE: When `githost` is not specified, the given certificate is used for all HTTPS repositories.
+
+// TODO
+// [discrete]
+// == Configuring support for self-signed Git repositories on OpenShift
+
+Create and start new workspace. Every container used by the workspace mounts a special volume that contains a file with the self-signed certificate. The `gitconfig` section contains information about the Git server host and the path to the certificate. For example:
```
Comment (rkratky) [inline src/main/pages/che-7/installation-guide/proc_deploying-che-with-support-for-git-repositories-with-self-signed-certificates.adoc]: @vparfonov, it wasn't clear to me what is "the gitconfig section" -- is that a part of some config file? Can we clarify that?

Comment (vparfonov) [reply src/main/pages/che-7/installation-guide/proc_deploying-che-with-support-for-git-repositories-with-self-signed-certificates.adoc]: @rkratky, added missed words

Comment (rkratky) [reply src/main/pages/che-7/installation-guide/proc_deploying-che-with-support-for-git-repositories-with-self-signed-certificates.adoc]: Got it, thanks, @vparfonov -- I didn't realize this was about the regular Git config :)

**Comment by rkratky** [comment]
@vparfonov, is this ready to be merged, or are you going to be adding the content for OpenShift, too?

**Comment by vparfonov** [comment]
@rkratky we can merge it and later I will add information about OpenShift in other PR after fixing https://github.com/eclipse/che/issues/15285

**Review by rkratky — APPROVED** [review]

**Pull request merged**